### PR TITLE
correct some translation suggested by @Emphasia

### PR DESCRIPTION
**Pull request by abnerlee**
Fix https://github.com/typora/Typora-i18n/issues/11 and https://github.com/typora/Typora-i18n/pull/10

### CODE REVIEW

File: zh-Hans.lproj/Front.strings [hunk 1]
```diff
@@ -39,12 +39,12 @@
...
 "Align Center" = "居中对齐";
 "Align Right" = "右对齐";
-"Resize Table" = "修改表格";
+"Resize Table" = "调整表格";
 "Delete Table" = "删除表格";
 
 /* Images */
 "Copy Image To..." = "将图片拷贝到...";
-"Typora are trying copy the newly inserted image to folder <code id='image-create-folder-target'></code> according to your YAML Front Matter setting. But the target folder does not exist, create it now?" = "Typora正在尝试将新插入的图片拷贝到YAML Front Matter中指定的文件夹<code id='image-create-folder-target'></code>， 但目标目录不存在，是否创建该文件夹？";
+"Typora are trying copy the newly inserted image to folder <code id='image-create-folder-target'></code> according to your YAML Front Matter setting. But the target folder does not exist, create it now?" = "TTypera试图拷贝新插入图片至YAML Front Matter中的指定目录<code id='image-create-folder-target'></code>， 但目标目录不存在，是否创建该文件夹？";
```
Comment (Emphasia) [inline zh-Hans.lproj/Front.strings]: TTypera?

File: zh-Hans.lproj/Front.strings [hunk 2]
```diff
@@ -63,11 +63,11 @@
 "Loading" = "加载中";
 "Selected folders contains too many files. \nPlease switch to <a id='switch-to-tree-on-oversize'>File Tree view</a> for better performance." = "文件过多，请切换到<a id='switch-to-tree-on-oversize'>文件树视图</a>。";
 "New File" = "新建文件";
-"Unpin Outline Panel" = "取消大纲视图";
+"Unpin Outline Panel" = "关闭大纲视图";
 "Open Folder..." = "打开文件夹...";
 "Action" = "操作";
 "Close Sidebar Menu" = "关闭";
-"Reveal in Finder" = "在Finder中显示";
+"Reveal in Finder" = "在访达中显示";
```
Comment (Emphasia) [inline zh-Hans.lproj/Front.strings]: what is "访达"..😂

File: zh-Hans.lproj/Menu.strings [hunk 3]
```diff
@@ -170,34 +170,34 @@
...
-"Insert Paragraph After" = "下方插入新段落";
-"Insert Paragraph Before" = "上方插入新段落";
+"Insert Paragraph After" = "下方插入段落";
+"Insert Paragraph Before" = "上方插入段落";
 "Delete Code Block" = "删除代码块";
 "Copy as MathML" = "复制MathML";
 "Copy as Tex" = "复制Tex代码";
 "Copy Math" = "复制公式";
-"Reveal in Finder" = "在Finder中显示";
+"Reveal in Finder" = "在访达中显示";
```
Comment (Emphasia) [inline zh-Hans.lproj/Menu.strings]: what is "访达"..😂

Comment (abnerlee) [reply zh-Hans.lproj/Menu.strings]: see https://sspai.com/post/41482

This will only be used under macOS. And we follow apple's translation rule 👐

Comment (Emphasia) [reply zh-Hans.lproj/Menu.strings]: In that case, "访达" does make sense. I'll pay attention to that in the future.

But from the reaction of "sspai" and other medias, in general, these translations have improvements, but there are some that not very accord with the expression of Chinese habit and make it difficult to understand need to be improved, such as "访达").
> 可系统名称不像宣传文案，大家吐槽一下就过去了，如果做不到准确易懂，甚至会影响到一些用户的使用体验，刚刚看到「访达（Finder）」「隔空投送（AirDrop）」这些词时，即使是老用户也难免需要反应一下，对此苹果表示他们对这次名称更改也很慎重，例如之前曾经出现在 macOS 系统更新中的「搓擦条（Multi-Touch Bar）」现在就更改为了更易理解的「触控条」。

Thus, for Typora's users, the original "Finder" (or, according to the current name in the system) might be more appropriate.
Just an idea.😁

Comment (abnerlee) [reply zh-Hans.lproj/Menu.strings]: Yeah, maybe we can postpone this translation, but if other apps use this word, we still will follow someday.

**Pull request merged**